flink-runtime-web/src/test/java/org/apache/flink/runtime/rest/compatibility/RestAPIStabilityTest.java

```diff
@@ -50,20 +51,21 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-/** Stability test and snapshot generator for the REST API. */
-final class RestAPIStabilityTest {
+/** Stability test and snapshot generator for the Runtime REST API. */
+public final class RestAPIStabilityTest {
```

Comment:
Let's move this class into flink-runtime (as a sort of `RestApiStabilityTestUtils`) and add a separate test for the runtime rest API that makes use of it similar to the sql gateway one.

This removes the need for the runtime-web test-jar.

Reply:
Done. Thank you @zentol.